plugins/module_utils/network/sonic/config/port_breakout/port_breakout.py

```diff
@@ -175,6 +176,56 @@ def _state_deleted(self, want, have, diff):
 
         return commands, requests
 
+    def _state_replaced(self, want, have, diff):
+        """ The command generator when state is replaced
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+
+        commands = diff
+        requests = self.get_modify_port_breakout_requests(commands, want)
+        if commands and len(requests) > 0:
+            commands = update_states(commands, "replaced")
+        else:
+            commands = []
+
+        return commands, requests
+
+    def _state_overridden(self, want, have, diff):
+        """ The command generator when state is overridden
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = []
+        requests = []
+        if not diff:
```

Comment:
This does not cover the case in which a subset of the existing breakout modes is specified in "want". If that occurs, "diff" will be empty and no action will occur. Instead, for this case, the breakout modes should be de-configured for all currently configured breakout interfaces except for the interface(s) specified in "want".

Please modify the logic here to handle this case.